includes/wikis.php

```diff
@@ -123,6 +123,12 @@
 		'theme-dark' => '#c6c6cA',
 		'api' => $baseurl . '/fifa/api.php',
 	],
+	'arenaofvalor' => [
```

Comment:
Wouldn't this need to go to the top of the list as a new wiki and have the new key added and set to true? Same for the COC one?